### ISSUE TITLE
Disable-on-submit/click button classes are not used for v3 questions

### ISSUE THREAD

**Issue by mwest1066** [PrairieLearn/PrairieLearn#1812]
We add classes `disable-on-submit` and `disable-on-click` to buttons in the question footer:

https://github.com/PrairieLearn/PrairieLearn/blob/master/pages/partials/questionFooter.ejs#L19

But the JS that uses these classes is only added for non-Freeform (that is, old-style v2) questions:

https://github.com/PrairieLearn/PrairieLearn/blob/master/pages/partials/questionFooter.ejs#L84

It looks like this might have been an error in the original implementation from Nov 2017, but since we haven't heard any complaints since then it's not clear that it's worth fixing?

**Comment by mwest1066** [comment]
Looks like we hit an actual consequence of this bug. Submissions `22446396` and `22446397` are identical (same student, same variant, same submitted answer) and were submitted within 0.2 seconds of each other. This was on an exam and caused the student to use up their only re-attempt at the question, so they lost points.

**Comment by tdy** [comment]
I can deal with this since I'm already patching `questionFooter.ejs`.

Sounds like we just need to copy the `script` up to the v3 block, correct?

How do we test it? Don't fully understand the underlying issue.

**Comment by mwest1066** [comment]
The underlying issue is that students sometimes manage to "double submit" by clicking the submit button very rapidly twice in a row, before the page has reloaded. I'd be happy with any other method of preventing this. I can't really remember why we chose this way of fixing it (with the JS disabling the buttons).

**Comment by mwest1066** [comment]
Testing is a bit tricky, I have to agree. I'm not sure how to do that unless someone is a very fast clicker 😄

Trying to double-submit with the current code would be a good start.

Alternatively, we could just activate this code again, test that regular submissions still work, and assume that everything is ok.

**Comment by tdy** [comment]
Got it, should be fixed now. Originally I could double-submit and now I can't (the disabling is also visibly happening "immediately").

Side questions about PL contribution guidelines (didn't see these mentioned in docs):

1. **Indents/whitespace:** I'm assuming we should leave formatting as is to reduce diff spam. It's bugging me that this file has 2-space and 4-space indentation mixed throughout. So far I've only cleaned up indents within lines that I was already patching (this was the policy in some other repos I've worked on).
2. **PR contents:** Should I submit this `disable-on-submit` patch and the `d-flex` patch as separate PRs? They both patch this file but are conceptually unrelated.
